home/migrations/0041_contentpage_whatsapp_template_lower_case_name.py

```diff
@@ -0,0 +1,46 @@
+# Generated by Django 4.1.13 on 2024-01-08 10:11
+
+from django.db import migrations, models
+
+
+def update_template_names(ContentPage, ContentType, Revision):
+    """
+     Update historic data template names to lower case
+    """
+    for page in ContentPage.objects.filter(is_whatsapp_template=True):
+        if not page.whatsapp_title:
+            continue
+        template_prefix = page.whatsapp_title.lower().replace(" ", "_")
+        page.whatsapp_template_name = f"{template_prefix}_{page.latest_revision.id}"
```

Comment:
I don't think we need to rename the templates, in fact that might lead to issues since they're already submitted to whatsapp. I think we just need to do what the `submit_whatsapp_template` function did, which was lowercase them, ie. `page.whatsapp_template_name = page.whatsapp_template_name.lower()`. (and the same for revisions).

I think then we'll also need to change the `if not page.whatsapp_title` check, since we now need to check if there's a `page.whatsapp_template_name`